label_studio/tests/data_import/test_uploader.py

```diff
@@ -70,7 +70,54 @@ def test_local_url_after_redirect(self, project, settings):
                 ValidationError
             ) as e:
                 load_tasks(request, project)
-            assert 'The provided URL was not valid.' in str(e.value)
+            assert 'URL resolves to a reserved network address (block: 127.0.0.0/8)' in str(e.value)
+
+        def test_user_specified_block(self, project, settings):
+            settings.SSRF_PROTECTION_ENABLED = True
```

Comment:
Do we have a test anywhere that outlines what the behaviour of this code is when: `settings.SSRF_PROTECTION_ENABLED = False`?

Reply:
It's not covered in depth, but the TL;DR is that we don't check for hosts that point to reserved addresses if SSRF protection is off. We do, however, reject urls with unsafe schemes like `file://`, as shown in this test: https://github.com/HumanSignal/label-studio/pull/5316/files#diff-140c3d60bd366710ad0470bbc1907a34d4f3065d1e8647dc33636d0268100bf1L38-L47